### PR TITLE
task of service content finished

### PR DESCRIPTION
**Pull request by GustavoUT22**
check it out!

### CODE REVIEW

File: src/pages/Services/ServicesContent/ServicesContent.styles.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import Container from 'ui/Container/Container'
+
+export const ContainerServices = styled(Container)`
+  gap: 3rem;
+
+  @media only screen and (min-width: ${breakpoints.md}) {
+    gap: 5.5rem;
+  }
+`
+interface ServiceContentWrapperProps {
```
Comment (CrhistianT7) [inline src/pages/Services/ServicesContent/ServicesContent.styles.ts:15]: this should be in types folders

File: src/pages/Services/ServicesContent/ServicesContent.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  ServiceLink,
+  ServiceLinksWrapper,
+} from './ServicesContent.styles'
+
+const ServicesContent: React.FC = () => {
+  return (
+    <Section type="margin" size="lg">
+      <ContainerServices size="xl" isfullwidth>
+        {ServicesData.map((service, index) => (
+          <ServiceContentWrapper key={service.id} index={index}>
```
Comment (CrhistianT7) [inline src/pages/Services/ServicesContent/ServicesContent.tsx:19]: Move all this to one component, and send the service as prop

File: src/pages/Services/ServicesData.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
+interface IService {
```
Comment (CrhistianT7) [inline src/pages/Services/ServicesData.ts:1]: move to types folder

File: src/pages/Services/ServicesData.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  name: string
+  to: string
+}
+
+interface IServiceCategory {
```
Comment (CrhistianT7) [inline src/pages/Services/ServicesData.ts:6]: move to types folder

Comment (GustavoUT22) [reply src/pages/Services/ServicesData.ts:6]: done

File: src/pages/Services/ServicesContent/ServicesContent.styles.ts (hunk 1 above)
Comment (GustavoUT22) [reply src/pages/Services/ServicesContent/ServicesContent.styles.ts:15]: done

File: src/pages/Services/ServicesContent/ServicesContent.styles.ts [hunk 5]
```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components'
+
+import { breakpoints } from 'shared/breakpoints'
+import { colors } from 'styles/colors'
+import { Link } from 'react-router-dom'
```
[the remark this answers is not in the document]
Comment (GustavoUT22) [reply src/pages/Services/ServicesContent/ServicesContent.styles.ts:5]: What component can I use instead of that?
ButtonLink maybe ?

**Pull request closed without merging**